HealthVault/Classes/MHVThingCache.h

```diff
@@ -0,0 +1,13 @@
+//
+//  MHVThingCache.h
```

Comment:
We should probably add this file and the .m to the 'Store' directory and not the root 'Classes' directory. Either that or create a new directory altogether (which might be a better idea). At some point we need to clean up our directory structure into public and private directories, so we don't expose internal functionalities of the framework to consumers.